apps/api/v2/src/ee/calendars/services/calendars.service.ts

```diff
@@ -42,11 +42,11 @@ export class CalendarsService {
       throw new NotFoundException("User not found");
     }
 
-    return getConnectedDestinationCalendars(
-      userWithCalendars,
-      false,
-      this.dbWrite.prisma as unknown as PrismaClient
-    );
+    return getConnectedDestinationCalendars({
```

Comment:
this is still  getConnectedDestinationCalendars  using instead of getConnectedDestinationCalendarsAndEnsureDefaultsInDb ?